config.toml

```diff
@@ -111,6 +111,38 @@ name = "Community"
 url = "https://github.com/helm/community"
 weight = 5
 
+# Korean
+# [languages.ko]
+# title = "Helm"
+# description = "Helm - The Kubernetes Package Manager."
+# contentDir = "content/ko"
+# languageName = "한국어 Korean"
+# weight = 1
+
+# [[languages.ko.menus.main]]
+# name = "Home"
+# url = "/"
+# weight = 1
+
+# [[languages.ko.menus.main]]
+# name = "Docs"
+# url = "/docs"
+# weight = 2
+
+# [[languages.ko.menus.main]]
+# name = "Charts"
+# url = "https://hub.helm.sh/"
+# weight = 3
+# external = "yes"
+
+#[[languages.zko.menus.main]]
+#name = "Blog"
+#url = "/blog"
+#weight = 2
+
```

Comment:
Should community be added for Korean and Chinese also?
```
[[languages.en.menus.main]]
name = "Community"
[...]
```

Reply:
Good idea. done.